### PR TITLE
Add npm as a deployment provider to travis

### PR DESCRIPTION
**Pull request by Elchi3**
I followed the steps here https://docs.travis-ci.com/user/deployment/npm/
(basically, I just did `$ travis setup npm`)

With this, everyone with push access should be able to push a new tag, for example like this:
```cli
$ npm version patch -m "12th alpha version"
$ git push --tags
```
And then this should: 
1) add a new commit to master updating the version in package.json,
2) add a new tag, like "v0.0.12" to the repo,
3) trigger travis to react on the new tag and to publish version 0.0.12 to npm.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+notifications:
+  email: false
+deploy:
+  provider: npm
+  email: mdn-npm@mozilla.com
+  api_key:
+    secure: tPV4Rve++cmJeYOEhB0/bYaUVIxx9fyKm3Nn4mEBoTuqUojxCzjYaRNqhCREUigQayKDd6Efa6tzHbTygyU41WUNRLtRYENqv060nj8Bdzx7bjgm8dRqdctrvgisfqBfuQa+5MbliZblhD6M7ZyoURqK2stY64JGhvonKG1w0zl2sVyhDNSzoJwx6dx57upkVvXFqk4PQtmtLxRc27YlEiPa7yUCkXYUVCrQVQst7iJfR3pOvvfsR3lhyCJqVrV7cFgTy1VmpxHK/PjbVMJx4JahLbDHbS1I+jRWwx8/aPKMuQfoDn+34Y6uqVe+NaAwoy7bppsKOx/hpAY9HNfLBjIk8oe2n1JXmQg9J8bQ4sIijn1uflcly+LD2FYJSqtkHSsTSTiGHSH6+0O79tYcFRp3E62Xq+ED4Bx4h44Il4W9Z9ggrU4Lcp5BzNGgwf76U6V/Gx/vgd4BmiyDV/7+1MrHJxM67P/5rNAD+fdYQ6Hh5d7765ljBT4ttokoyhgKhEA5ShjPVt+ubotbgJ7ixY72O926qEAvQ7n74dEd4vnGXp2cO/h2dEPAstENY1xeuMyT4fz2CZ/zCr6xEZ9vkevPSFXoHdk2BzrFKLhelBZw9VqmD5RSwhR4/6dB4VxMWB79QACdHslMCWkFVgiFWTuAa4Prmd7J6E5B03dtHd4=
+  on:
+    tags: true
+    branch: master
```
Comment (jwhitlock) [inline .travis.yml]: I believe ``branch: master`` says to release with each merge to master. Please remove this line and just leave the ``tags`` one.

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+language: node_js
+node_js: stable
+sudo: false
+notifications:
+  email: false
+deploy:
+  provider: npm
+  email: mdn-npm@mozilla.com
+  api_key:
+    secure: tPV4Rve++cmJeYOEhB0/bYaUVIxx9fyKm3Nn4mEBoTuqUojxCzjYaRNqhCREUigQayKDd6Efa6tzHbTygyU41WUNRLtRYENqv060nj8Bdzx7bjgm8dRqdctrvgisfqBfuQa+5MbliZblhD6M7ZyoURqK2stY64JGhvonKG1w0zl2sVyhDNSzoJwx6dx57upkVvXFqk4PQtmtLxRc27YlEiPa7yUCkXYUVCrQVQst7iJfR3pOvvfsR3lhyCJqVrV7cFgTy1VmpxHK/PjbVMJx4JahLbDHbS1I+jRWwx8/aPKMuQfoDn+34Y6uqVe+NaAwoy7bppsKOx/hpAY9HNfLBjIk8oe2n1JXmQg9J8bQ4sIijn1uflcly+LD2FYJSqtkHSsTSTiGHSH6+0O79tYcFRp3E62Xq+ED4Bx4h44Il4W9Z9ggrU4Lcp5BzNGgwf76U6V/Gx/vgd4BmiyDV/7+1MrHJxM67P/5rNAD+fdYQ6Hh5d7765ljBT4ttokoyhgKhEA5ShjPVt+ubotbgJ7ixY72O926qEAvQ7n74dEd4vnGXp2cO/h2dEPAstENY1xeuMyT4fz2CZ/zCr6xEZ9vkevPSFXoHdk2BzrFKLhelBZw9VqmD5RSwhR4/6dB4VxMWB79QACdHslMCWkFVgiFWTuAa4Prmd7J6E5B03dtHd4=
```
Comment (jwhitlock) [inline .travis.yml]: I was surprised this was inserted into .travis.yml - I was expecting a server-side secret, such as https://www.npmjs.com/package/travis-deploy-example. But, this is what ``travis setup npm`` spits out, so I assume it is correct.

Comment (Elchi3) [reply .travis.yml]: I've added NPM_TOKEN as an environment variable to the travis repo settings now.

File: .travis.yml (hunk 1 above)
Comment (Elchi3) [reply .travis.yml]: Thanks, I think I misunderstood the docs.

**Comment by jwhitlock** [comment]
Thanks for the updates! Let's try it for the Monday update 😬

**Pull request merged**